hashtables/ex2/ex2_tests.py

```diff
@@ -15,7 +15,7 @@ def test_short_case(self):
         expected = ["PDX", "DCA"]
         result = reconstruct_trip(tickets, 3)
 
-        self.assertEqual(expected, result)
```

Comment:
Hi @tolaked, please revert the changes made to this test file. Changes are not to be made to the test file. Thanks 😄